ontology/yaml/resources/EMERGENCY_MONITORING/entity_types/LDS.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+LDS_STD:
+  description: "Basic leak detection device."
+  is_canonical: true
+  implements:
+  - LDS
+  uses:
+  - water_leak_alarm
+
+LDS_BCM:
+  description: "Leak sensing device with broken cable status. "
+  is_canonical: true
+  implements:
+  - LDS
+  uses:
+  - water_leak_alarm
+  - broken_cable_alarm
+
+LDS_BC2X:
+  description: "Leak sensing device with broken cable status. "
```

Comment:
nit: empty space after .
It looks ugly when generated to rdf.

Reply:
fixed